src/views/simplified-account/settings/webhooks/index.njk

```diff
@@ -1,9 +1,75 @@
+{% from "./_webhook_status.njk" import webhookStatusTag %}
 {% extends "../settings-layout.njk" %}
 
+{% macro webhookSubscriptions(webhook) %}
+  <ul class="govuk-list">
+    {% for subscription in webhook.subscriptions | sort %}
+      {% set key = subscription | upper %}
+      <li>{{ eventTypes[key] }}</li>
+    {% endfor %}
+  </ul>
+{% endmacro %}
+
 {% block settingsPageTitle %}
   Webhooks
 {% endblock %}
 
 {% block settingsContent %}
-<h1 class="govuk-heading-l">Webhooks</h1>
+  <h1 class="govuk-heading-l">Webhooks</h1>
+  <p class="govuk-body">Use webhooks to send a message to your application when a payment is processed.</p>
+  <p class="govuk-body">Find out more about
+    <a class="govuk-link" href="https://docs.payments.service.gov.uk/webhooks/">setting up webhooks in your application</a>.
+  </p>
+
+
+  {{ govukButton({
+    text: "Create a new webhook",
+    classes: "govuk-button",
+    href: createWebhookLink
+  }) }}
+
+  {% for webhook in webhooks %}
```

Comment:
In the design there is a line which says either "There are no configured webhooks" or "Configured webhooks (<some number>)" depending on whether there are webhooks configured. The design shows two webhooks but says "Configured webhooks (1)" so I'm not sure whether this refers to the number of active configured webhooks, or if it should be "Configured webhooks (2)"

Reply:
oops - big oversight there, well spotted, will sort that out.

Reply:
Following discussion with designers, I've split the headings into active and deactivated and applied the counts accordingly.